src/Common/Parameters/Parameters.jl

```diff
@@ -0,0 +1,17 @@
+"""
+    Parameters
+
+Module containing sets of parameters.
+"""
+module Parameters
+
+export AbstractParameterSet
+abstract type AbstractParameterSet{FT<:AbstractFloat} end
+
+include("PlanetParameters.jl")
```

Comment:
```suggestion
include("Planet.jl")
```
is fine, since it's in the `Parameters` directory.

Reply:
Also the files under `CLIMA/parameters` and `CLIMA/src/Common/Parameters` should mirror each other (same names, same order etc)